http-client/src/test/java/com/facebook/airlift/http/client/AbstractHttpClientTest.java

```diff
@@ -621,6 +622,7 @@ public void testResponseStatusCode()
         assertEquals(statusCode, 543);
     }
 
+    /*
     @Test
     public void testResponseStatusMessage()
```

Comment:
Delete this instead of commenting it out

Reply:
Done